app/controllers/redirect_controller.rb

```diff
@@ -39,21 +39,28 @@ def is_datastream
     return (/^DS\d+/ =~ params[:ds]) != nil
   end
 
-  def find_item
+  def find_item_id
     uuid = params[:uuid].split(":")[1]
-    file = GenericFile.find(fedora3uuid: uuid).first
```

Comment:
It doesn't need to strip out the "uuid:" prefix for this solr query to work on the backend